_extras/guide.md

```diff
@@ -174,3 +174,7 @@ $ export LANGUAGE=en_US.UTF-8
 [lesson-example]: https://github.com/carpentries/lesson-example/
 [makefile2graph]: https://github.com/lindenb/makefile2graph
 [zipfile]: {{ page.root }}/files/make-lesson.zip
+
```

Comment:
Could you move the whole new section up, so that the links appear last?

Reply:
done